port/action/schema.go

```diff
@@ -499,7 +498,7 @@ func StringPropertySchema() schema.Attribute {
 			},
 		},
 		"encryption": schema.StringAttribute{
-			MarkdownDescription: "The algorithm to encrypt the property with",
+			MarkdownDescription: "The algorithm to encrypt the property with. Accepted value: `aes256-gcm`, `aes128-gcm`",
```

Comment:
same here

Reply:
only `aes256-gsm` is currently supported for property encryption